.github/workflows/run_game.yaml

```diff
@@ -23,16 +18,39 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        # Names of agents which the main script should be run with
-        agents: ['agent_1', 'agent_2']
+        # Configurations with which the script will be run
+        agents: ['agent_1', 'agent_2'] # Add the name of your agent here
+        levels: [60]
+        starting_hp: [10]
+        starting_attack: [100]
+        starting_shield: [100]
+        base_stamina: [400]
+        threshold_pct: [0.6]
+        agent_random_quantity: [25, 100]
+
     name: Testing performance of ${{ matrix.agents }}
     steps:
       - uses: actions/checkout@v3
       - name: Setup go
         uses: actions/setup-go@v3
         with:
           go-version-file: './pkg/infra/go.mod'
-      - name: Test performance of agent
-        run: go run main.go ${{ matrix.agents }}
-        working-directory: .
-        continue-on-error: false
+
+      - name: Build main.go to tmp directory
+        run: go build -o /tmp/infra
+        working-directory: pkg/infra
+
+       - name: Set environment variables
```

Comment:
Couldn't this be done using [this](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs) and [this](https://docs.github.com/en/actions/learn-github-actions/environment-variables)

Reply:
Good point